images/router/haproxy/reload-haproxy

```diff
@@ -12,8 +12,13 @@ if [ -f $pid_file ]; then
   old_pid=$(<$pid_file)
 fi
 
+# Assumes that the docker ip is the first entry in the hosts file.
+# This port is non-configurable via the router cmd, it is HAProxy specific.  It should
+# match the port used in the haproxy-config.template
+peer_name=$(awk 'NR==1 {print $1}' /etc/hosts):1937
```

Comment:
hmm, actually, I will need think about this.  This is not the actual binding, it is telling HAProxy what it should consider the local name in lieu of the default host name.  It uses this to match the peer section to know what is 'me' during synchronization (and opens the ip/port that the label matches).  Since I create the peer section based on the endpoints of the service the names are just the ip/ports, hence using the ip/port as the "name" with `-L`.  

For example, rather than ip/port the peer section could've looked like:

```
peers openshift_peers
    peer thisisme 0.0.0.0:1937
    peer notme 1.1.1.1:1937

and on the haproxy command

-L thisisme
```

So perhaps the right solution is using the pod downward api to get the ip address, match that to the port and use a way to label the peer correctly (using ip/port for all notme servers).  Then I can remove the items in the shell script and just run with -L <known static label>